src/modules/org.junit.platform.launcher/module-info.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+module org.junit.platform.launcher {
+	requires transitive org.junit.platform.commons;
+
+	exports org.junit.platform.launcher;
+	exports org.junit.platform.launcher.core;
+	exports org.junit.platform.launcher.listeners;
+	exports org.junit.platform.launcher.tagexpression;
```

Comment:
:x: The last one only contains internal classes and should not be exported.

Reply:
Will be removed.